src/Calculator/Views/MainPage.xaml

```diff
@@ -163,16 +162,31 @@
                HorizontalOffset="0"
                IsLightDismissEnabled="False"
                LightDismissOverlayMode="Off"
-               Opened="Popup_Opened"
-               VerticalOffset="{x:Bind AppTitleBar.Height, Mode=OneWay}">
+               Opened="Popup_Opened">
             <Popup.ChildTransitions>
                 <TransitionCollection>
                     <EntranceThemeTransition/>
                 </TransitionCollection>
             </Popup.ChildTransitions>
 
             <Grid x:Name="PopupContent" x:Load="false">
-                <local:Settings/>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="32"/>
```

Comment:
The height of the title bar row here is 32 while the Auto height of the title bar row in TitleBar.xaml would be 20 with your change.

It would be better to define the title bar height in the application resource to make the grid row height here align with the row height in TitleBar.xaml.

Reply:
Suggest binding the row height with title bar height.

Reply:
And please search "PopupContent.Height" in Mainpage.xaml.cs, you will see some other lines about setting the popup height, you need to update accordingly.

Reply:
Changes in the latest commit:
 - Move the back-button to settings control.
 - Bind the height of back-button with title bar
 - Remove unused items.